<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aspectsDialog</class>
 <widget class="QDialog" name="aspectsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="aspectoLabel">
       <property name="text">
        <string>Aspecto:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="aspectoComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="planeta1Label">
       <property name="text">
        <string>Planeta 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="planeta1ComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="planeta2Label">
       <property name="text">
        <string>Planeta 2:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="planeta2ComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="gradeGroupBox">
       <property name="title">
        <string>Grados:</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="gradosLabel">
            <property name="text">
             <string>Valor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="gradosSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="toleranciaLabel">
            <property name="text">
             <string>Tolerancia:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="toleranciaGradosSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="minutesGroupBox">
       <property name="title">
        <string>Minutos:</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="valorLabel">
            <property name="text">
             <string>Valor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="minutosSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="toleranciaLabel_2">
            <property name="text">
             <string>Tolerancia:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="toleranciaMinutosSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="secondsGroupBox">
       <property name="title">
        <string>Segundos:</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="valorLabel_2">
            <property name="text">
             <string>Valor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="segundosSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="toleranciaLabel_3">
            <property name="text">
             <string>Tolerancia:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="toleranciaSegundosSpinBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>aspectsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>aspectsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gradeGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gradosSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gradeGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>toleranciaGradosSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minutesGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minutosSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minutesGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>toleranciaMinutosSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>secondsGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>segundosSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>secondsGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>toleranciaSegundosSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
